packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -129,7 +132,7 @@ export const ChatComposite = (props: ChatCompositeProps): JSX.Element => {
             onRenderTypingIndicator={onRenderTypingIndicator}
             onRenderMessage={onRenderMessage}
             onFetchParticipantMenuItems={onFetchParticipantMenuItems}
-            {...fileSharingOptions()}
+            {...fileSharingOptions}
```

Comment:
nit: Instead of this being memoized and spread across multiple lines, can this just be:
```suggestion
            /* @conditional-compile-remove(file-sharing) */
            fileSharing={options?.fileSharing}
```

Reply:
(`options` is already passed into ChatScreen, this prop may not be necessary at all)

Reply:
Good catch, will update with the proposed code! I would prefer not to add it to `options` as options is a public ChatCompositeOption type